scripts/upgrade

```diff
@@ -22,7 +22,7 @@ fi
 #=================================================
 ynh_script_progression "Upgrading source files..."
 
-ynh_setup_source --dest_dir="$install_dir/app" --full_replace --keep="app/data/_data_/_default_/configs/application.ini"
```

Comment:
why are we removing `/app` path ?
if the path is `$install_dir/app/data/_data_/_default_/configs/application.ini` we should keep `/app`, no?

Reply:
According to the commit message, `keep` path is relative to `dest_dir`, not `install_path`